libImaging/Resample.c

```diff
@@ -184,7 +184,6 @@ precompute_coeffs(int inSize, int outSize, struct filter *filterp,
                 } else if (x == xmax - 1) {
                     xmax -= 1;
                 }
-                continue;
             }
             k[x] = w;
```

Comment:
Is this bug a case of using uninitialized memory?  It kind of looks that way from the non-deterministic pattern of the alpha in the bug.

Would initializing all the coefficients to 0 have prevented this?